Application/Services/TaskService.cs

```diff
@@ -57,13 +57,13 @@ public async Task<TaskResponse> CreateAsync(CreateTaskRequest createTaskRequest)
     {
         logger.LogInformation("Started creating task from Service Layer");
         await createTaskValidator.ValidateAndThrowAsync(createTaskRequest);
-        if (!await projectRepository.IsProjectExistsAsync(createTaskRequest.ProjectId))
+        if (!await repository.IsEntityExistsAsync(createTaskRequest.ProjectId))
```

Comment:
it doesn't work, you're trying to get a project using task repo, you need 2 different repositories for this (task and project)

Reply:
So i need to get my repositories back>????)

Reply:
of course not, you can inject two generic repo with different T

Reply:
oh okay!